.scalafmt.conf

```diff
@@ -14,8 +14,7 @@ project.excludeFilters = [
   sbt-test
   contrib/package.scala
   scalameta/semanticdb-scalac/src/main/resources/scalac-plugin.xml
-  scalameta/semanticdb-scalac/src/main/scala-2.11.11/scala/meta/internal/SemanticdbAnalyzer.scala
-  scalameta/semanticdb-scalac/src/main/scala-2.12.3/scala/meta/internal/SemanticdbAnalyzer.scala
```

Comment:
Did I use the exclude syntax incorrectly here? Is scalafmt skipping these files anyway?

Reply:
Oooooh, this explains a lot.